src/constants/index.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import { GelatoAddressBook } from "../types";
 
+export const W3F_API_ENDPOINT = "https://api.gelato.digital/w3f/networks";
```

Comment:
I think we need to have both prod & dev:
```
export const W3F_API_ENDPOINT = "https://api.gelato.digital/w3f/networks";
export const W3F_API_ENDPOINT_DEV = "https://api.dev.gelato.digital/w3f/networks";
```

So that we can switch depending if we are using dev mode or not